src/main/java/org/junit/runners/Parameterized.java

```diff
@@ -23,34 +23,55 @@
  * When running a parameterized test class, instances are created for the
  * cross-product of the test methods and the test data elements.
  * <p>
- * For example, to test a Fibonacci function, write:
+ * For example, this class has a method that adds two integers and returns the sum.
  * <pre>
+ * &#047;* This is the class being tested. *&#047;
+ * public class IntAdder {
+ *     public static int add(int addend1, int addend2) {
+ *         return addend1 + addend2;
+ *     }
+ * }
+ * </pre>
+ * The following is a test for the above example class.
+ * For each element returned from the <code>&#064;Parameters</code> annotated method,
+ * an instance of the <code>Parameterized</code> test class is created and
+ * its <code>&#064;Test</code> annotated method (e.g. <code>test()</code>) is invoked.
+ * <pre>
+ * &#047;* This is is the unit test for {&#064;link IntAdder}. *&#047;
  * &#064;RunWith(Parameterized.class)
- * public class FibonacciTest {
- *     &#064;Parameters(name= &quot;{index}: fib[{0}]={1}&quot;)
+ * public class IntAdderTest {
+ *     &#047;* Each of these is used to create a new {&#064;link IntAdderTest} instance. *&#047;
+ *     &#064;Parameters(name= &quot;{index}: {0} + {1} = {2}&quot;)
```

Comment:
style-nit: our new style is to put spaces before and after the `=` (see https://google-styleguide.googlecode.com/svn/trunk/javaguide.html)